build-tools/src/main/java/org/elasticsearch/gradle/testclusters/StandaloneRestIntegTestTask.java

```diff
@@ -42,23 +42,6 @@ public abstract class StandaloneRestIntegTestTask extends Test implements TestCl
     private boolean debugServer = false;
 
     public StandaloneRestIntegTestTask() {
-        Spec<Task> taskSpec = t -> getProject().getTasks()
```

Comment:
Why was this removed? We need to make sure we never cache multi-phase tests like many of the BWC tests.

Reply:
This logic was moved out of this constructor into LegacyRestTestBasePlugin. That ensures when using StandaloneRestTestPlugin or others we still apply this logic. We do not need it for our new test framework